Angular/src/app/admin-table/admin-table.component.spec.ts

```diff
@@ -1,14 +1,31 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { AdminTableComponent } from './admin-table.component';
+import { NavbarComponent } from '../navbar/navbar.component';
+import { NgbAlertModule } from '@ng-bootstrap/ng-bootstrap';
+import { DataService } from '../_services/data.service';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AuthGuardComponent } from '../auth-guard/auth-guard.component';
+import { AdminAuthGuardComponent } from '../admin-auth-guard/admin-auth-guard.component';
+import { CookieService } from 'ngx-cookie-service';
 
 describe('AdminTableComponent', () => {
   let component: AdminTableComponent;
   let fixture: ComponentFixture<AdminTableComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ AdminTableComponent ]
+      declarations: [ AdminTableComponent,
+                      NavbarComponent
+                    ],
+     imports: [ NgbAlertModule,
+                RouterTestingModule
+              ],
+     providers: [ AuthGuardComponent,
+                  AdminAuthGuardComponent,
+                  DataService,
+                  CookieService
+                ]
```

Comment:
same here, " providers: [ AuthGuardComponent, AdminAuthGuardComponent, DataService, CookieService] " - example of standard format.

Reply:
I prefer the formatting like this: https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17